domain/schema/model/sql/0022-resource.sql

```diff
@@ -133,3 +134,13 @@ CREATE TABLE unit_resource (
     REFERENCES unit (uuid),
     PRIMARY KEY (resource_uuid, unit_uuid)
 );
+
```

Comment:
**suggestion** Add a comment which states why this table even exists (to not store password in objectstore, which is plaintext). 

It is an interesting information to keep, because we may want to rollback it whenever we may have a more secure/trusted way to deal with it.

IMHO, resources CAN already contains password that users may want not to push in objectstore (if it is not trusted enough), since it is basically configuration.  So it seems quite probable that a solution will arise in the future and this part of the solution may be required to align in term of consistency.

Reply:
We should document this, but I wouldn't do it in the SQL. I do not expect folks to be reading the SQL files often. When @Aflynn50 writes the domain code it should be documented there. Perhaps around the domain type to read/write the data. 

A new core/objectstore/doc.go file would be perfect.

Reply:
Fair enough